configure

```diff
@@ -242,6 +279,17 @@ EOF
   else
     echo '#define VA_LIST_IS_ARRAY 1'
   fi
+
+  cc_header_file HAVE_PTY_H 'pty.h'
+  cc_header_file HAVE_LIBUTIL_H 'libutil.h'
+  cc_header_file HAVE_UTIL_H 'util.h'
+
+  cc_statement HAVE_STAT_TV_NSEC \
```

Comment:
I'll leave a TODO in here to investigate further for now.

Here's what I can guess:
- `pty.h` and alternatives seem important to OSH since that's related to pseudoterminals.
- [`HAVE_STAT_TV_NSEC`](https://github.com/aykamko/oil/blob/babc25c6532e98003d116b07a49477327cdd3bae/Python-2.7.13/Modules/posixmodule.c#L1467-L1515) is relevant to the `stat()` filesystem function, seems important too but maybe not